authorize.py

```diff
@@ -57,9 +57,25 @@ def authorize(event,context):
     ret_val = { "statusCode":200,"isBase64Encoded": False, "headers": { "Content-Type":"application/json" },"body" : json.dumps(update_val)}
     return config.add_cors_headers(ret_val)
 
+#NOT A LAMBDA
+def authorize_then_consume(event, context):
+    rv = authorize(event, context)
+    if 'link' in event:
+        consumption_event = {
+            'link': event['link'],
+            'token': json.loads(rv['body'])['auth']['token']
+        }
+        consume_val = consume.consumeUrl(consumption_event, None)
+        if consume_val['statusCode'] != 200:
+            rv['statusCode'] = 400
+    return rv
+
+
 def create_user(event, context, mlh = False):
     if not config.is_registration_open():
```

Comment:
Would people be able to create acc on day of if magic links?

Reply:
You can always do it w/ magic links.

I generally reopen registration on the day-of too.